File: add/codesnippet/CSharp/e-system.windows.forms.f_19_1.cs
private void Form1_HelpButtonClicked(Object sender, CancelEventArgs e) {

System.Text.StringBuilder messageBoxCS = new System.Text.StringBuilder();
messageBoxCS.AppendFormat("{0} = {1}", "Cancel", e.Cancel );
messageBoxCS.AppendLine();
MessageBox.Show(messageBoxCS.ToString(), "HelpButtonClicked Event" );
}

File: add/codesnippet/VisualBasic/e-system.windows.forms.f_19_1.vb
Private Sub Form1_HelpButtonClicked(sender as Object, e as CancelEventArgs) _ 
     Handles Form1.HelpButtonClicked

    Dim messageBoxVB as New System.Text.StringBuilder()
    messageBoxVB.AppendFormat("{0} = {1}", "Cancel", e.Cancel)
    messageBoxVB.AppendLine()
    MessageBox.Show(messageBoxVB.ToString(),"HelpButtonClicked Event")

End Sub